src/com/carolinarollergirls/scoreboard/defaults/DefaultScoreBoardModel.java

```diff
@@ -231,6 +265,31 @@ public void setTimeoutType(String owner, boolean review) {
 			}
 			setTimeoutOwner(owner);
 			setOfficialReview(review);
+			if (!settings.getBoolean(SETTING_STOP_PC_ON_TO)) {
+				boolean stopPc = false;
+				if (!owner.equals(TIMEOUT_OWNER_NONE)) {
```

Comment:
This doesn't handle when the timeout type is unknown, the clock won't be stopped in that case.